ComputeClient/Compute.Contracts/Requests/Server20/ServerListOptions.cs

```diff
@@ -117,10 +117,20 @@ public sealed class ServerListOptions : FilterableRequest
 		/// </summary>
 		public const string ReplicationTargetDatacenterIdField = "replicationTargetDatacenterId";
 
-		/// <summary>
-		/// The "deploymentMode" field name.
-		/// </summary>
-		public Guid? TagKeyId { get; set; }
+        /// <summary>
+        /// The "sourceType" field name.
+        /// </summary>
+        public const string SourceTypeField = "sourceType";
+
+        /// <summary>
+        /// The "sourceValue" field name.
+        /// </summary>
+        public const string SourceValueField = "sourceValue";
```

Comment:
There isn't an explicit ordering (probably not intentional) with the existing filter fields in here so have just added the new filters to the end but can move them if we think its better to have them somewhere else in this class. 

Prefer to keep the change focused rather than fixing the ordering for the existing parameters.